tools/stress-cluster/services/Stress.Watcher/src/PodEventHandler.cs

```diff
@@ -215,5 +228,78 @@ public bool ShouldStartPodChaos(WatchEventType type, V1Pod pod)
 
             return true;
         }
+
+        public async Task DeleteResources(V1Pod pod)
+        {
+            if (!ShouldDeleteResources(pod))
+            {
+                Logger.Debug($"Skipping pod deletion for pod {pod.Name()}.");
```

Comment:
Minor change: it's resource deletion, not pod deletion, and the pod name is already in the logger properties.

```suggestion
                Logger.Debug($"Skipping resource deletion.");
```